build_tools/scripts/git/check_submodule_init.py

```diff
@@ -29,24 +29,16 @@ def run():
 
   output = os.popen("git submodule status")
   submodules = output.readlines()
-  runtime_submodules = [
-      "third_party/benchmark", "third_party/cpuinfo", "third_party/flatcc",
-      "third_party/googletest", "third_party/liburing", "third_party/libyaml",
-      "third_party/musl", "third_party/spirv_cross",
-      "third_party/spirv_headers", "third_party/tracy",
-      "third_party/vulkan_headers", "third_party/vulkan_memory_allocator",
-      "third_party/webgpu-headers"
-  ]
+
+  with open(
+      os.path.dirname(os.path.realpath(__file__)) + "/runtime_submodules.txt",
+      "r") as f:
+    runtime_submodules = f.read().split("\n")
 
   for submodule in submodules:
-    if (submodule.strip()[0] == "-"):
-      if (args.runtime_only and submodule.split()[1] in runtime_submodules):
-        print(
-            "The git submodule '%s' is not initialized. "
-            "Please run `./build_tools/scripts/git/update_runtime_submodules.sh`"
-            % (submodule.split()[1]))
-        sys.exit(1)
-      elif (not args.runtime_only):
+    if submodule.strip()[0] == "-":
+      if (args.runtime_only and
+          submodule.split()[1] in runtime_submodules) or not args.runtime_only:
```

Comment:
I got distracted talking to Stella when writing this suggested edit and didn't finish cleaning up the conditional :-) This can be simplified

```suggestion
    prefix = submodule.strip()[0]
    name = submodule.split()[1]
    if prefix == "-" and (not args.runtime_only or name in runtime_submodules):
```

Reply:
Done.